libs/providers/flagsmith/README.md

```diff
@@ -0,0 +1,93 @@
+# Flagsmith Provider
+
+This provider is an implementation for the [JavaScript SDK](https://docs.flagsmith.com/clients/javascript/) of [Flagsmith](https://flagsmith.com).
```

Comment:
I think this is definitely up to the folks at Flagsmith, but I agree. It would be best to make this very clear to consumers this is for web.

I would recommend you use the same pattern for the dirs/filenames as well.

Reply:
I did consider this but the way the provider works would also work for react-native.

Reply:
Client could also be fine if this is more the thing you want to be showing of :)

Reply:
flagsmith-client it is 😃